src/styles/markdown/base.scss

```
@@ -138,13 +138,13 @@ body.dark,
 		font-size: 0.8em;
 		font-weight: 500;
 
-		color: var(--secondary_on-variant);
+		color: var(--secondary_on-low);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)